python/hail/contrib/ldsc.py

```diff
@@ -0,0 +1,156 @@
+
```

Comment:
I'm a bit worried about the name clash from calling this `ldsc`. maybe `ldscore` to make it totally clear?